lib/FusionInventory/Agent/Inventory.pm

```diff
@@ -70,6 +70,7 @@ my %fields = (
     PHYSICAL_VOLUMES => [ qw/DEVICE PV_PE_COUNT PV_UUID FORMAT ATTR
                              SIZE FREE PE_SIZE VG_UUID/ ],
     PORTS            => [ qw/CAPTION DESCRIPTION NAME TYPE/ ],
+    POWERSUPPLIES    => [ qw/PARTNUM SERIAL VENDOR POWER IS_ATX/ ],
```

Comment:
About the specs, could you better replace:
    SERIAL by SERIALNUMBER
    VENDOR by MANUFACTURER
    POWER by POWER_MAX
You should also add : NAME and MODEL

Reply:
Also you should remove IS_ATX as the agent don't have to assume the power supply is not ATX if it can find the information. This problem should better be handled on the server-side.